06_modules_lab/03_bonus.py

```diff
@@ -1,13 +1,40 @@
-""" Write a program that searches current working directory
-for files larger than 1MB. Every time you find such a file print
-its name to the user.
-
-- When the program finds a large file. It should ask the user
-  a message asking if she wants to delete it, and delete the
-  file if requested
-
-- Take threshold and path as command line arguments
-
-Bonus: Use argparse module to parse command line arguments
-"""
-
+#!/usr/bin/env python
+
+__author__ = 'Eabel'
+
+import sys
+import os
+import argparse
+
+def Usage():

```

Comment:
argparse already prints a nice help + usage message when script is used incorrectly